addons/website_sale/controllers/main.py

```diff
@@ -1102,6 +1108,129 @@ def address(self, **kw):
         render_values.update(self._get_country_related_render_values(kw, render_values))
         return request.render("website_sale.address", render_values)
 
+    @http.route(
+        ['/shop/express'],
+        type='json',
+        methods=['POST'],
+        auth="public",
+        website=True,
+        sitemap=False,
+    )
+    def process_express_checkout(self, **post):
```

Comment:
You can unpack `billing_address` directly in the function here, it will fail when calling it without `billing_address` but it's already the case:
```suggestion
    def process_express_checkout(self, billing_address, **post):
```